src/main/java/com/gmail/goosius/siegewar/SiegeController.java

```diff
@@ -391,35 +394,31 @@ public static void setSiege(Town town, boolean bool) {
 	}
 
 	/**
-	 * This method returns true
-	 * - The given town is in a nation, and
-	 * - One (or more) of the nation's home towns is a siege defender
-	 *
 	 * @param town the town to check
-	 * @return true if one (or more) of the nation's home towns is a siege defender
+	 * @return true if the town has a nation & that nation is fighting a home-defence war
+	 *
+	 * Note: A home defence war is when one or more of the nation's home towns is under siege.

```

Comment:
Something that could replace the term "home towns" could be "natural towns".

Reply:
Ok, updating terminology